__tests__/get.test.js

```diff
@@ -209,6 +209,40 @@ describe("GET /api/articles (with query parameters)", () => {
 	});
 });
 
+describe("GET /api/articles (topic filtering)", () => {
+	test("GET:200 articles accept filtering by topic", () => {
+		return request(app)
+			.get("/api/articles?topic=mitch")
+			.expect(200)
+			.then(({ body: { articles } }) => {
+				expect(articles.length).toBe(12);
+				articles.forEach((article) => {
+					expect(article).toHaveProperty("topic", "mitch");
+				});
+			});
+	});
```

Comment:
This is great TDD 🏆 you're only checking the `topic` property has the correct value of `'mitch'` because the other properties are being covered by another previous test 👍 

It looks like this describe block is sitting in the global scope of the file, one thing I would suggest for good organisation of test blocks is that it could be nested inside of the describe block you have for the other tests on this endpoint to keep it all together